examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
I would rather not add a new example, but replace NiftiMasker + SVC in many examples by the decoder.